### ISSUE TITLE
Starting apps from scratch VS restarting one app

### ISSUE THREAD

**Issue by alemoi** [Starcounter/Starcounter#3873]
Assuming the database is running. We have 3 apps A, B, C.
What should be faster in calendar time:
1. Codehost is empty. Start all apps sequentially: A, B, C.
or
2. All apps are running in codehost (A, B, C). Then start one already running app, lets say A (which causes all apps restart).

My initial guess was that 1 would take less time, because we don't restart the codehost there. But seems its not the same in practice as we tried with @miyconst 

@per-samuelsson do you have any theoretical idea why that can be?

**Comment by alemoi** [comment]
https://github.com/Starcounter/Starcounter/issues/3481

**Comment by per-samuelsson** [comment]
> @per-samuelsson do you have any theoretical idea why that can be?

Not a clue I'm afraid. Option 1 should be the fastest for sure. But that assumes you don't include bootstrapping the host within the timed scope.

To be clear what I mean

```
// Start overall timer
staradmin kill all
staradmin start db
// Start app bootstrapping timer
star a.cs
star b.cs
star c.cs
// Stop both timers
```

If 1 is "app bootstrapping timer", 1 should be faster than 2.

**Comment by alemoi** [comment]
> If 1 is "app bootstrapping timer", 1 should be faster than 2.

Yes, we will double check this. But so far we were reproducing exactly like this. Except that we used .exe instead of .cs

**Comment by per-samuelsson** [comment]
@alemoi 

Can you read through this issue https://github.com/Starcounter/CompanyTrack/issues/254 and see if it relates to this one?
